index.html

```diff
@@ -7,7 +7,7 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <h1>HELLO WORLD!</h1>
+    <h1 style="color: green">HELLO WORLD!</h1>
```

Comment:
Good addition, however, I would consider moving this to an isolated style sheet for clarity.